cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/ProgramRuntimeProvider.java

```diff
@@ -47,7 +47,7 @@
    * The execution mode of the program runtime system.
    */
   enum Mode {
-    LOCAL, DISTRIBUTED
+    IN_MEMORY, LOCAL, DISTRIBUTED
```

Comment:
`LOCAL` means non distributed. Why need to have a `IN_MEMORY`? What's the difference between `IN_MEMORY` and `LOCAL`?